src/mono/wasm/runtime/loader/blazor/WebAssemblyResourceLoader.ts

```diff
@@ -15,12 +16,12 @@ export class WebAssemblyResourceLoader {
 
     private cacheLoads: { [name: string]: LoadLogEntry } = {};
 
-    static async initAsync(bootConfig: BootJsonData, startOptions: Partial<WebAssemblyStartOptions>): Promise<WebAssemblyResourceLoader> {
+    static async initAsync(bootConfig: BootJsonData, loadBootResource?: LoadBootResourceCallback): Promise<WebAssemblyResourceLoader> {
```

Comment:
could we drop the class on this PR already ?

Reply:
We could, but I would rather do it in follow up. I want to deliver these changes to the blazor. The important thing is that it not used in blazor.